package.json

```diff
@@ -10,9 +10,13 @@
     "@testing-library/user-event": "^14.1.1",
     "emailjs": "^4.0.1",
     "firebase": "^9.12.1",
+    "i18next": "^22.0.3",
+    "i18next-browser-languagedetector": "^7.0.0",
+    "i18next-http-backend": "^2.0.0",
```

Comment:
we dont really need the http-backend package. We can keep it if they were installed via the command automatically though. So no worries on that. But you installed them with npm, we dont use npm. We will add it via yarn. So please uninstall the i18next dependency with `npm uninstall` and add it back again with yarn. If yarn command will give you problems, install yarn by running `npm install -g yarn` then add the i18next dependency with yarn.